### DISCUSSION TITLE
VaporFile 'root' support

### DISCUSSION THREAD

**Discussion by lonnylot** [laravel/nova-issues#5694] [Ideas & Feature Requests]
Vapor will statically upload to `/tmp` of the provided bucket.

However, if you set the `root` key in `filesystems.disks.s3` then the `VaporFile` field will try to copy from the prefixed path instead of the root `/tmp`

Is it possible to support either:

a) Uploading to the `root` prefixed `tmp` folder
b) Copy from the `/tmp` directory of the bucket without the `root`?
c) Support the documented scoped filesystem (https://laravel.com/docs/10.x/filesystem#scoped-and-read-only-filesystems)

**Comment by crynobone** [comment]
This needs to be supported by Vapor first before it can be done in Laravel Nova.
